ai/core/src/unitycatalog/ai/core/oss.py

```diff
@@ -122,12 +129,14 @@ def __init__(self, api_client: ApiClient, **kwargs: Any) -> None:
                 "The 'api_client' must be an instance of unitycatalog.client.ApiClient"
             )
 
-        self.uc = FunctionsApi(api_client=api_client)
+        self.functions_client = FunctionsApi(api_client=api_client)
```

Comment:
Can we create a separate class like this:
```
class UnityCatalog:
    functions = FunctionsApi(...)
    catalogs =  CatalogsApi(...)
    schemas = SchemasApi(...)
```

So that we only need to create self.uc = UnityCatalog(api_client) here

Reply:
This makes a LOT of sense :D I'll update!